crypto/zk/utils.go

```diff
@@ -20,6 +20,19 @@ const (
 	publicSigLen = 7
 )
 
+var modulus, _ = new(big.Int).SetString("21888242871839275222246405745257275088548364400416034343698204186575808495617", 10)
```

Comment:
seeing this number made me 21888242871839275222246405745257275088548364400416034343698204186575808495617 times curious to know where does it come from. please add a comment?

Reply:
hahaha, for sure, thanks a lot!